pyproject.toml

```diff
@@ -77,7 +77,7 @@ django-haystack = "*"
 citeproc-py = "*"
 pyswot = "*"
 django-redis = "*"
-django-allauth = {extras = ["socialaccount"], version = "*"}
+django-allauth = {version = "*", extras = ["mfa", "socialaccount"]}
```

Comment:
The lockfile has changed on main so you will need to merge/rebase and generate the lockfile again.